src/main.js

```diff
@@ -1 +1,20 @@
+import {render, RenderPosition} from './render.js';
+import {templateHeaderSeaction,price} from './view/trip-info.js';
```

Comment:
Лучше так не делать, лучше оставить всю отрисовку внутри, просто получаем вьюшку инфы и отрисуем её
```suggestion
import {InfoView} from './view/info-view.js';
```